src/gui/widgets/mapper_proxystyle.cpp

```diff
@@ -36,6 +37,29 @@
 
 namespace OpenOrienteering {
 
+namespace  {
+
+/**
+ * This function recognizes dock widget related widgets by the class name
+ * matching "DockWidget".
+ * 
+ * This function helps to customize the style for classes like
+ * "QDockWidgetTitleButton" or "MapEditorDockWidget".
+ */
+bool isDockWidgetRelated(const QWidget* widget)
```

Comment:
This causes an unused-function warning on all systems but Android. I will have to add `#ifdef Q_OS_ANDROID`

Reply:
Added `Q_DECL_UNUSED` instead.